src/content/blog/2024-08-05-rafiki-code-architecture.mdx

```diff
@@ -0,0 +1,203 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Architecture Breakdown of Rafiki (Pending Title)"
+description: 
+date: 2024-08-05
+slug: rafiki-low-level-intro
+author: Nathan Lie
+author_url: https://www.linkedin.com/in/nathan-lie-138a73121
+tags:
+  - Interledger
+  - Rafiki
+---
+import LargeImg from '/src/components/pages/LargeImg.astro'
+
+## Introduction
+
+It has been said that the Interledger Foundation upholds open technology as one of its core values. It has also been said that [Rafiki](https://github.com/interledger/rafiki), a white-label application that enables Interledger usage, upholds this value by being open-source and allowing anyone to contribute.
+
+It has _not_ been said, however, how one comes to intimately understand Rafiki. It's not impossible, to be sure, but the Interledger endeavor has been around for long enough that a write-up at a lower level is warranted.
+
+So, what makes Rafiki tick? What do we see when we venture beyond _what_ Rafiki does and into _how_ it does those things? 
+
+### Disclaimer
+This article assumes the reader has high-level knowledge of the following concepts:
+
+* [Open Payments](https://openpayments.dev/introduction/overview/)
+* [Interledger](https://interledger.org/developers/get-started/#how-does-interledger-work)
+* [Rafiki](https://rafiki.dev/introduction/overview/)
+
+## Components
+
+Rafiki is comprised primarily of three packages:
+
+* A [`backend`](https://github.com/interledger/rafiki/tree/main/packages/backend) package that extends an API for managing Open Payments resources like Incoming or Outgoing Payments. 
+* An [`auth`](https://github.com/interledger/rafiki/tree/main/packages/auth) package that provide third parties with a method of acquiring authorization to manage Open Payments resources on the Rafiki instance's associated `backend` package.
+* A [`frontend`](https://github.com/interledger/rafiki/tree/main/packages/frontend) that serves as an Admin-level UI for that Rafiki instance. It allows the manager of that Rafiki instance to directly manage items such as other peers on the Interledger network or what currencies it supports.
+
+### The `backend` and `auth` stacks
```

Comment:
would it be good to include the other packages in there? Maybe we can add a table of contents in here and just one sentence explain what they are? (and how they are managed with pnpm?)